### ISSUE TITLE
Update the Project Community Standard

### ISSUE THREAD

**Issue by LaansDole** [miketvo/imdupes#52]
@miketvo The project is still missing a few templates for issue, code of conduct and security, do you want me to do it for you and you can review it later?

**Comment by miketvo** [comment]
Yes please @LaansDole, thank you for bringing this up. I was so busy that I didn't get the chance to update these. So just take your time whenever you're free.

I am assigning you to this issue now.
